core/src/scene/sceneLoader.h

```diff
@@ -43,15 +43,14 @@ class SceneLoader {
     /* loads a texture with default texture properties */
     void loadTexture(const std::string& url, Scene& scene);
     void loadMaterial(YAML::Node matNode, Material& material, Scene& scene);
+    MaterialTexture loadMaterialTexture(YAML::Node matCompNode, Scene& scene);
     void loadShaderConfig(YAML::Node shaders, Style& style, Scene& scene);
     SceneLayer loadSublayer(YAML::Node layer, const std::string& name, Scene& scene);
-    MaterialTexture loadMaterialTexture(YAML::Node matCompNode, Scene& scene);
     Filter generateAnyFilter(YAML::Node filter, Scene& scene);
     Filter generateNoneFilter(YAML::Node filter, Scene& scene);
     Filter generatePredicate(YAML::Node filter, std::string _key);
 
-    // Style Mixing helper methods
-    YAML::Node mixStyle(const Mixes& mixes);
+    std::unordered_set<std::string> m_mixedStyles;
```

Comment:
Looking ahead to the scene-reloading PRs (in which all of `SceneLoader`'s functions become static) it would make sense to ensure that `m_mixedStyles` will behave correctly in that case (or, better yet, remove the need for it at all)